quine/src/main/scala/com/thatdot/quine/app/ingest/KafkaSrcDef.scala

```diff
@@ -56,6 +57,16 @@ object KafkaSrcDef {
 
     val keyDeserializer: ByteArrayDeserializer = new ByteArrayDeserializer() //NO-OP
 
+    // Create Seq of kafka properties: combination of user passed properties from `kafkaProperties`
+    // as well as those templated by `KafkaAutoOffsetReset` and `KafkaSecurityProtocol`
+    // NOTE: This diveragance between how kafka properties are set should be resolved, most likely by removing
+    // `KafkaAutoOffsetReset`, `KafkaSecurityProtocol`, and `KafkaOffsetCommitting.AutoCommit`
+    // in favor of `KafkaIngest.KafkaProperties`. Additionally, the current "template" properties override those in kafkaProperties
+    val properties = kafkaProperties.map(_.toSeq).getOrElse(Seq.empty[(String, String)]) ++ Seq(
```

Comment:
There's an overload of `.withProperties` that takes a `Map[String, String]` - then this could be left in terms of `Map`s, and we wouldn't have to pass in a `Seq` via varargs... also then duplicates would be resolved up front, and would still have the same "last write wins" behavior - i.e. ` Map(1 -> "foo") ++ Map(1 -> "bar")` == `Map(1 -> "bar")`

Reply:
thanks for pointing that out. Still newish to Scala, ill make the change